_posts/2020-06-29-release-1.11.0.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post 
+title:  "Apache Flink 1.11.0 Release Announcement" 
+date: 2020-06-29T08:00:00.000Z
+categories: news
+authors:
+- morsapaes:
+  name: "Marta Paes"
+  twitter: "morsapaes"
+
+excerpt: The Apache Flink community is proud to announce the release of Flink 1.11.0! This release marks the first milestone in realizing a new vision for fault tolerance in Flink, and adds a handful of new features that simplify (and unify) Flink handling across the API stack. In particular for users of the Table API/SQL, this release introduces significant improvements to usability and opens up completely new use cases, including the much-anticipated support for Change Data Capture (CDC)! A great deal of effort has also gone into optimizing PyFlink and ensuring that its functionality is available to a broader set of Flink users.
+---
+
+The Apache Flink community is proud to announce the release of Flink 1.11.0! This release marks the first milestone in realizing a new vision for fault tolerance in Flink, and adds a handful of new features that simplify (and unify) Flink handling across the API stack. In particular for users of the Table API/SQL, this release introduces significant improvements to usability and opens up completely new use cases, including the much-anticipated support for Change Data Capture (CDC)! A great deal of effort has also gone into optimizing PyFlink and ensuring that its functionality is available to a broader set of Flink users.
```

Comment:
Do we know if this release is special regarding typical release metrics, such as # of contributors, # of commits, # of tickets resolved?

If it is, I would include them into this section of the release announcement. It could help people come up with "clickbait-y" tweets and headlines about the release, ultimately helping our attention.

Reply:
Will check — only included the number of contributors in the last section, this time. I have to admit it was a deliberate decision, as my own words felt "clickbait-y" to me when re-reading the last announcement. 😬

Reply:
I was thinking about including this in the Community Update following the release instead.

Reply:
So you have these numbers? Are they exciting compared to the other recent releases?

Reply:
It's pretty much the same baseline as 1.10, regarding tickets and contributors. I can include it